state.go

```diff
@@ -519,12 +519,12 @@ func (s *State) MessageAdd(message *Message) error {
 }
 
 // MessageRemove removes a message from the world state.
-func (s *State) MessageRemove(message *Message) error {
+func (s *State) MessageRemove(channelID, messageID string) error {
```

Comment:
Arghh sorry to be a butthole dude. But I didn't want to change the existing API.